src/pages/mypage/settings.tsx

```diff
@@ -1,14 +1,38 @@
 import Link from 'next/link';
+import { useRouter } from 'next/router';
 import styled from 'styled-components';
 
-// 개인정보 설정 페이지
+import { requestError } from '@@types/shared';
 import Header from '@components/Header';
+import { NETWORK_ERROR, UNKNOWN_ERROR } from '@constants/error';
+import { requestLogout } from '@request';
+import { showAlertModal } from '@utils/modal';
 
 const Settings = () => {
+  const router = useRouter();
+
+  const handleLogOut = async () => {
```

Comment:
```suggestion
  const handleLogout = async () => {
```

logout은 같은 문맥의 단어라 굳이 camelCase로 작성안하셔도 될 것 같습니다!

Reply:
오호 꼼꼼한 리뷰 감사합니다!